dev/com.ibm.ws.jca.cm/resources/com/ibm/ws/j2c/resources/J2CAMessages.nlsprops

```diff
@@ -336,6 +336,16 @@ SHAREDPOOL_REMOVESHAREDCONNECTION_ERROR_J2CA1003=J2CA1003E: Failed to remove MCW
 SHAREDPOOL_REMOVESHAREDCONNECTION_ERROR_J2CA1003.explanation=The attempt to remove an MCWrapper failed, causing a potential problem in the shared pool.
 SHAREDPOOL_REMOVESHAREDCONNECTION_ERROR_J2CA1003.useraction=CMSG0002
 
+# --------------------------------
+RA_FREE_CONNECTION_ERROR_J2CA1004=J2CA1004I: The Connection Manager received a fatal connection error from the Resource Adapter for resource {1}. The connections state is: {2}. The exception is: {0}
+RA_FREE_CONNECTION_ERROR_J2CA1004.explanation=The connection manager has been notified by the resource adapter that an error has occurred on the connection. This informational message is logged so that the user can see the exception that was returned.
```

Comment:
```suggestion
RA_FREE_CONNECTION_ERROR_J2CA1004.explanation=The connection manager was notified by the resource adapter that an error occurred on the connection. This message is logged so that the user can see the exception that was returned.
```